examples/churn_prediction/Build and Deploy Churn Ensemble.ipynb

```diff
@@ -36,7 +36,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 45,
```

Comment:
Wish there was a better way to ignore notebooks so that when I run them they don't appear in git diffs. i'm not sure if this exists :)